src/main/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidator.java

```diff
@@ -11,37 +12,38 @@
 import javax.xml.validation.Schema;
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
-import java.io.File;
+import java.io.IOException;
 
+import tds.exam.results.configuration.ExamResultsTransmitterServiceProperties;
 import tds.exam.results.trt.TDSReport;
 import tds.exam.results.validation.TDSReportValidator;
 
 @Component
 public class XSDBackedTDSReportValidator implements TDSReportValidator {
     private final JAXBContext jaxbContext;
     private final Validator validator;
+    private final ExamResultsTransmitterServiceProperties properties;
 
-    @Autowired
-    public XSDBackedTDSReportValidator(final JAXBContext jaxbContext) {
+    XSDBackedTDSReportValidator(final JAXBContext jaxbContext,
+                                @Value("classpath:xsd/TestResultsTransmissionFormat_Schema.xsd") final Resource xsd,
```

Comment:
This is very much a WIP application.  We're adding pieces of the TRT slowly over time instead of one single large PR with the whole system written.